services/frontend-service/src/ui/NavigationBar/NavigationBar.scss

```diff
@@ -13,6 +13,20 @@
   }
 
   .mdc-drawer__content {
-    height: 100%;
+    height: 95%;
+  }
+
+  .mdc-list-item-indicator {
+    background-color: transparent;
+    width: 5%;
```

Comment:
In general, anything that's more specific or not intuitive like `100%` should go into a variable. 
Please take a look here: https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/WORKSPACE_DOCUMENT/-Mn5HLuHv2wjoq0QKPax/

Reply:
Totally agreed! Already done it.